dask/bytes/local.py

```diff
@@ -87,10 +116,25 @@ def open_files(path):
             for _path in filepaths]
 
 
+def write_files(paths, mode='wb'):
+    """ Open many files for writing.  Return delayed objects.
+
+    See Also
+    --------
+    dask.bytes.core.write_files: User function
+    """
+    myopen = delayed(open)
+    files = [myopen(_path, mode=mode,
+                   dask_key_name='write-%s' % tokenize(_path, time.time()))
+            for _path in paths]
+    return files
```

Comment:
Should we return files or should we consume sequences and write those sequences to a file, returning `None` or something?